<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="aClose"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolPrimary">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aProveedores"/>
   <addaction name="aClientes"/>
   <addaction name="aGastos"/>
   <addaction name="separator"/>
   <addaction name="aContabilidad"/>
  </widget>
  <widget class="QToolBar" name="toolSecondary">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="aNuevo"/>
   <addaction name="aEditar"/>
   <addaction name="aVer"/>
  </widget>
  <action name="aNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="aProveedores">
   <property name="text">
    <string>Proveedores</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="aEditar">
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="aClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
   <property name="toolTip">
    <string>Clientes</string>
   </property>
  </action>
  <action name="aClose">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aGastos">
   <property name="text">
    <string>Gastos</string>
   </property>
   <property name="toolTip">
    <string>Gastos</string>
   </property>
  </action>
  <action name="aVer">
   <property name="text">
    <string>Ver</string>
   </property>
   <property name="toolTip">
    <string>Ver</string>
   </property>
  </action>
  <action name="aContabilidad">
   <property name="text">
    <string>Contabilidad</string>
   </property>
   <property name="toolTip">
    <string>Contabilidad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
